eeraster.py

```diff
@@ -283,17 +290,42 @@ def dem():
       'projects/sat-io/open-datasets/GLO-30').select("b1"))
   return _dem
 
+def _download_raster(raster_name: str):
+  image = ee.Image(
+            os.path.join(
+              eeddf.ee_reference_rasters_path() +
+              f"{raster_name}"))
+  task = ee.batch.Export.image.toDrive(
+    image=image,
+    description=raster_name,
```

Comment:
You should set `dimensions`, `projection`, and `scale` so that they are consistent across every raster

https://developers.google.com/earth-engine/apidocs/export-image-todrive

Reply:
Ack, updated these based on offline conversation.